.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
```

Comment:
unfortunately we can't remove the Node v4 tests for v1.x of the module.

Reply:
Webpack v4 does not support node 4.  We could continue to keep the `devDependency` at webpack v3.x for the sake of these tests though.

All I really wanted to do is update the `peerDependency` to allow for v4.x.

Reply:
Well in fairness, you can't remove a node dependency in a patch for major version that previously supported it. That's a massive semver violation, and that's why v2.x dropped support for it. Node 4 will have to stay and you'll have to find a way to not run webpack v4 tests on that node version. These are the things that bite you when trying to update an old version instead of upgrading.

Reply:
We aren’t actually removing support for node 4 with this change.  We just wouldn’t be running tests against it (because that is not possible with webpack v4).  If we kept webpack v3 as the `devDependency` version we could keep running tests against node v4.

Really this situation isn’t dissimilar from what you’re already practicing.  Webpack-dev-middleware technically supports webpack v1 and v2 but no tests are run for these versions... ¯\_(ツ)_/¯